config/application.rb

```diff
@@ -1,11 +1,15 @@
-require File.expand_path('../boot', __FILE__)
+require_relative 'boot'
 
+require "rails"
 # Pick the frameworks you want:
 require "active_model/railtie"
+require "active_job/railtie"
 require "active_record/railtie"
+require "active_storage/engine"
 require "action_controller/railtie"
 require "action_mailer/railtie"
-# require "action_view/railtie"
+require "action_view/railtie"
```

Comment:
Do we need action_view for Rails API projects?

Reply:
no. removed it